### PR TITLE
🏗 Report Travis job/build URLs to test case reporting

### PR DESCRIPTION
**Pull request by rcebulko**
Oddly enough, there is no way to link to a Travis job/build log with just the repo and job/build number.

### CODE REVIEW

File: build-system/common/travis.js [hunk 1]
```diff
@@ -48,131 +48,98 @@ function isTravisPushBuild() {
...
+    if (!testFn()) {
+      log(red('ERROR:'), errorMsg, 'Cannot get', cyan(`process.env.${envKey}`));
+    }
+    return process.env[envKey];
+  };
+}
+
+const travisBuildEnv = (envKey) =>
+  travisEnv(isTravisBuild, 'This is not a Travis build.', envKey);
+
```
Comment (danielrozenberg) [inline build-system/common/travis.js:70]: Nice!

**Review by danielrozenberg — APPROVED** [review]

File: build-system/common/travis.js [hunk 2]
```diff
@@ -48,131 +48,98 @@ function isTravisPushBuild() {
...
-  if (!isTravisBuild()) {
-    log(
-      red('ERROR:'),
-      'This is not a Travis build. Cannot get',
-      cyan('process.env.TRAVIS_BUILD_NUMBER') + '.'
-    );
-  }
-  return process.env.TRAVIS_BUILD_NUMBER;
-}
+const travisBuildNumber = travisBuildEnv('TRAVIS_BUILD_NUMBER');
```
Comment (rsimha) [inline build-system/common/travis.js:75]: With this refactor, `travisBuildNumber`, etc. have all changed from a function to a constant, and should no longer have function-style JSDoc (with the `@return`, etc.).

(The same thing applies to all the `timedExec*` constants (formerly functions) in `exec.js` that were refactored in #30118.)

**Review by rsimha — COMMENTED** [review]
Nice PR overall! One post-merge comment below. Can be done at leisure in a future cleanup, no hurry.

File: build-system/common/travis.js (hunk 2 above)
Comment (rcebulko) [reply build-system/common/travis.js:75]: My bad, I saw this PR turn green and I didn't see your comment until after the merge. Sorry bout that! Fixing up with `@function` annotation now

Comment (rcebulko) [reply build-system/common/travis.js:75]: (GitHub AJAX updates don't hit all parts of the UI, annoyingly)

Comment (rcebulko) [reply build-system/common/travis.js:75]: https://github.com/ampproject/amphtml/pull/30556

**Pull request merged**